runelite-client/src/main/java/net/runelite/client/plugins/poh/BurnerOverlay.java

```diff
@@ -61,47 +58,55 @@ public BurnerOverlay(Client client, PohConfig config, PohPlugin plugin)
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		if (!config.showBurner())
+		if (!config.showBurnerTime())
 		{
 			return null;
 		}
-
+		//Get each object in POH
 		plugin.getPohObjects().forEach((object, tile) ->
 		{
+			//If planes match
 			if (tile.getPlane() == client.getPlane())
 			{
-				if (BURNER_UNLIT.contains(object.getId()))
-				{
-					drawBurner(graphics, "Unlit", object, Color.RED);
-				}
-				else if (BURNER_LIT.contains(object.getId()))
+				if (getBURNER_LIT().contains(object.getId()))
```

Comment:
You should probably inverse like this entire method, all these nested ifs are horrible, just do early returns instead